src/common.js

```diff
@@ -397,3 +410,17 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+exports.getProcessEnv = function () {
+  var env = objectAssign({}, process.env);
+
+  if (config.nodepath) {
+    env[PATH] += (process.platform === 'win32' ? ';' : ':') + path.join(process.cwd(), 'node_modules', '.bin') + (process.platform === 'win32' ? ';' : ':');
```

Comment:
Could you change this to [path.delimiter](https://nodejs.org/api/path.html#path_path_delimiter)?

Also, I don't think we always want a trailing delimiter, so you can get rid of the second `(process.platform === 'win32' ? ';' : ':')`. Please correct me if Windows differs from Unix on this point.